### PR TITLE
Changes current and live for .NET Client docs

### PR DESCRIPTION
**Pull request by szabosteve**
## Overview

This PR changes the `current` branch and the `live` branches for the .NET docs. As the 8.0 client is not GA now, it might be confusing for the users to expose 8.1+ versions for the docs. For this reason, this PR sets `current` to 8.0 and stops exposing branches newer than 8.0 in `live`. When the client versioning and stack versioning are aligned, we start exposing those branches again.

DO NOT MERGE BEFORE https://github.com/elastic/elasticsearch-net/pull/6658

### CODE REVIEW

**Comment by szabosteve** [comment]
@elasticmachine test this please

**Comment by szabosteve** [comment]
@elasticmachine test this please

**Review by gtback — APPROVED** [review]
LGTM (once the other PR is merged and the tests pass 😉 )

**Pull request merged**
